components/operator/Dockerfile

```diff
@@ -19,12 +19,12 @@ RUN go mod download
 COPY components/operator components/operator
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o operator ./components/operator/main.go
+RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -o operator ./components/operator/main.go
```

Comment:
Shouldn't we add ARG for TARGETOS and TARGETARCH?

Reply:
Indeed, we should. Thanks for catching this

Reply:
done